docs/DeveloperGuide.adoc

```diff
@@ -1507,6 +1520,197 @@ Use case ends.
 +
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: Add a note to a module
+
+*Pre-condition*
+
+Module must exist in data file.
+
+*MSS*
+
+1.  Teacher wants to add a note to a module.
+2.  System displays message of successful addition of a note to the module.
```

Comment:
Based on Steph's comments, it would be better to write the MSS the following way:

```
1. Teacher wants to add a note to a module.
2. System adds the note to the module.
3. System displays a message confirming the successful addition of the note.
```

Note that this way explicitly mentions the actual adding of the note, as well as displaying the confirmation message.

Other MSS's with the system doing an action followed by displaying a confirmation message should follow this approach.

Reply:
Try to adopt an alternating flow between the system and the role. I recommend you merge lines 2 and 3 into a single line.

Reply:
Changed as requested. I combined steps 2 and 3 from your suggestion into a single step to adhere to the alternating flow between user and system.